nifi-nar-bundles/nifi-confluent-platform-bundle/nifi-confluent-schema-registry-service/src/main/java/org/apache/nifi/confluent/schemaregistry/ConfluentSchemaRegistry.java

```diff
@@ -166,6 +202,21 @@ public void onEnabled(final ConfigurationContext context) {
             }
         }
 
+        final boolean authTypeSet = validationContext.getProperty(PROP_AUTH_TYPE).isSet();
+        final boolean authUsernameSet = validationContext.getProperty(PROP_BASIC_AUTH_USERNAME).isSet();
+        final boolean authPasswordSet = validationContext.getProperty(PROP_BASIC_AUTH_PASSWORD).isSet();
+        if (authTypeSet) {
+            if (!authUsernameSet || !authPasswordSet) {
```

Comment:
See comments on the Authentication Type property and recommend reviewing and integrating dependsOn property support if needed, which might remove the need for some of this custom validation.

Reply:
@pvillard31 can you help with this one?